autocomplete/complete.fish

```diff
@@ -1,15 +1,15 @@
-complete -c tldr -xf -s v -d "print verbose output"
-complete -c tldr -xf -l version -d "print version and exit"
-complete -c tldr -xf -s h -l help -d "print this help and exit"
-complete -c tldr -xf -s u -l update -d "update local database"
-complete -c tldr -xf -s c -l clear-cache -d "clear local database"
-complete -c tldr -rf -f -s p -l platform -a "linux osx sunos common" -d "select platform, supported are linux / osx / sunos / common"
-complete -c tldr -r -s r -l render -a PATH -d "render a local page for testing purpose"
+complete -c tldr -f -s v -d "print verbose output"
+complete -c tldr -x -l version -d "print version and exit" -n "__fish_use_subcommand"
```

Comment:
I used quotes around `__fish_use_subcommand` because most other examples I found used quotes around the `-n` commands. This is probably only because other fish commands use arguments, however. I tested the autocompletion without quotes and it still works. Thank you for suggesting this!